two-node-cluster/3-node-cluster.yml

```diff
@@ -523,7 +592,7 @@ Parameters:
     NoEcho: true
   DBInstanceType:
     Type: String
-    Default: db.m4.xlarge
+    Default: db.m4.2xlarge
```

Comment:
Why we are using a differebt type of instance to this VM and seems the change has done to this VM only

Reply:
It's only the database instance (RDS DB Instance).
We have used [db.m4.2xlarge](https://aws.amazon.com/rds/instance-types/) for all the other deployment patterns (Single node, 2 nodes) as well.